pyenv-win/libexec/pyenv-update.vbs

```diff
@@ -293,13 +293,13 @@ Sub main(arg)
             UpdateDictionary installers1, ScanForVersions(link.href, optIgnore, pageCount)
     Next
 
-    ' Now remove any duplicate versions that have the web installer (it's prefered)
+    ' Now remove any duplicate versions that have the offline installer (it's prefered)
     Dim minVers
     Dim fileName, fileNonWeb
     Dim versPieces
     Dim installers2
     Set installers2 = CopyDictionary(installers1) ' Use a copy because "For Each" and .Remove don't play nice together.
-    minVers = Array("2", "4", "", "", "", "", "", "")
+    minVers = Array("2", "4", "", "", "", "", "", "", "")
     For Each fileName In installers1.Keys()
         ' Array([filename], [url], Array([major], [minor], [path], [rel], [rel_num], [x64], [webinstall], [ext]))
```

Comment:
Is commentary line correct? New `[arm]` element was missed between `[x64]` and `[webinstall]` or not?

Reply:
I missed that, thanks.